### PR TITLE
Implement Router

### PR DESCRIPTION
**Pull request by principlebrothers**
### Develop a full website by creating three (3) pages: 
- Home, 
- Calculator, and 
- Quote.

### Style the page according to the Wireframe provided

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,12 +1,24 @@
...
 
 class App extends React.Component {
   render() {
     return (
       <div className="App">
-        <Calculator />
+        <Router>
+          <Nav />
+          <Routes>
+            <Route path="/home" element={<Home />} />
```
Comment (omarsalem7) [inline src/App.js]: - [ ] It will be better if you change the path from `/home` to `/`, so it will be the default page when the app starts.

**Review by omarsalem7 — CHANGES_REQUESTED** [review]
Hi @principlebrothers 😊, 

Good job so far! 👏
You have done well reaching this point of the milestone, but there are some issues that you still need to work on to go to the next project but you are almost there! 

you-are-near

## Highlights
- Descriptive pull request ✔️
- No linter errors ✔️
- Descriptive commit messages ✔️

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.js (hunk 1 above)
Comment (principlebrothers) [reply src/App.js]: Hi, @omarsalem7, Thank you for the reviewed feedback. I have implemented your suggestion.

**Review by prolajumokeoni — APPROVED** [review]
Hi @principlebrothers, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
## Highlights
- No linter errors ✅ 
- Git flow was used correctly 🚀 
- Good Pr title and description 💪🏿 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by principlebrothers** [comment]
I appreciate the feedback.

**Pull request merged**